src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,82 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_LOGIN_PASSWORD_LENGTH = 6;
+    private static final int MIN_USER_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
+    private boolean userAlreadyExist(User user) {
+        if (user != null && storageDao.get(user.getLogin()) != null) {
+            throw new InvalidDataException("User already have an account.");
+        }
+        return false;
+    }
+
+    private boolean loginQualityCheck(User user) {
```

Comment:
```suggestion
    private boolean isLoginValid(String login) {
```
you don't need whole user to validate his login